tasks/main.yaml

```diff
@@ -10,6 +10,14 @@
     uid: "{{ podman_user_uid }}"
     group: "{{ podman_user_group_name }}"
 
+- name: Setup authorized keys for podman user
+  authorized_key:
+    user: "{{ podman_user_name }}"
+    key: "{{ authorized_key }}"
+    path: '/home/{{ podman_user_name }}/.ssh/authorized_keys'
```

Comment:
You should only use `path` when `manage_dir` is false, otherwise we can make mistakes easily, so just remove `path` and use the default value.